native/cocos/editor-support/spine-wasm/spine-skeleton-instance.h

```diff
@@ -62,7 +63,10 @@ class SpineSkeletonInstance {
     std::vector<SpineDebugShape> &getDebugShapes();
     void resizeSlotRegion(const std::string &slotName, uint32_t width, uint32_t height, bool createNew = false);
     void setSlotTexture(const std::string &slotName, uint32_t index);
-
+    bool isCache = false;
+    bool enable = true;
+    float dtRate = 1;
+    bool isDelete = false;
```

Comment:
Could these variables be private?

And please take care of alignment and use `{}` to initialize member variables.

```c++
float dtRate{1.F};
bool isCache{false};
bool enable{false};
bool isDelete{false};
```

Reply:
The state of the parameter here needs to be passed through to the TS layer so public is used.